flink-filesystems/flink-s3-fs-base/pom.xml

```diff
@@ -46,14 +46,14 @@ under the License.
 			<scope>provided</scope>
 		</dependency>
 
-		<!-- Hadoop's file system abstraction (bundled) -->
-		<dependency>
-			<groupId>org.apache.flink</groupId>
-			<artifactId>flink-fs-hadoop-shaded</artifactId>
-			<version>${project.version}</version>
-		</dependency>
-
-		<!-- The Hadoop file system adapter classes (bundled) -->
+        <!-- Hadoop's file system abstraction (bundled) -->
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-common</artifactId>
```

Comment:
Along with a whoooole bunch of exclusions that were present.

Reply:
> what's this change about? If the s3 filesystems no longer depend on `flink-fs-hadoop-shaded`, why are we touching `flink-fs-hadoop-shaded` at all?
> Is `flink-fs-hadoop-shaded` even useful on it's own? If not (i.e., it always being bundled by some other functional component) then the module itself could probably just be removed.

You are right that the changes to `flink-fs-hadoop-shaded` should be reverted. If we unshade all modules, it can be completely removed.

Reply:
> Along with a whoooole bunch of exclusions that were present.

Also good catch, the final jar of the plugins is much bigger.